k8s/namespaces.go

```diff
@@ -72,8 +72,14 @@ func (kc *KubeClient) GetNamespaces(namespaceNames []string) ([]*corev1.Namespac
 // Any errors will be returned immediately and the remaining namespaces will be
 // skipped.
 func (kc *KubeClient) DeleteNamespaces(namespaceNames []string) error {
+	policy := metav1.DeletePropagationForeground
+	gracePeriod := int64(0)
```

Comment:
Do we maybe want a small grace period? Would this lead to a lot of containers not being able to shut down properly? Not sure if that is an issue, just wondering though.

Reply:
sounds good, will use 45 seconds

Reply:
done